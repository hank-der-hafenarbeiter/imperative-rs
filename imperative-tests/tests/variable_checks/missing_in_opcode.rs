use imperative_rs::InstructionSet;

#[derive(InstructionSet)]
enum Is {
    #[opcode = "0xffff"]
    Instr{w:u8},
}

fn main() {}
