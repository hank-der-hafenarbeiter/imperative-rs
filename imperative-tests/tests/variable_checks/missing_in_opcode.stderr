error: Variable w declared but never used in opcode.
 --> $DIR/missing_in_opcode.rs:6:11
  |
6 |     Instr{w:u8},
  |           ^
